Classes/JTLocalizeUtils.m

```diff
@@ -88,7 +94,12 @@ + (void)setLocalizationBundleToPath:(NSString *)bundlePath stringsTableName:(NSS
     }
     [self instance].localizationBundle = bundle;
     [self instance].effectiveLocale = [[bundle preferredLocalizations] firstObject] ?: @"en";
-    
+    if (![[[JTGameConfig sharedInstance] settingForKey:@"dutchEnabled" defaultValue:@(false)] boolValue]) {
+        if ([[self instance].effectiveLocale isEqualToString:@"nl"]) {
+            [self instance].effectiveLocale = @"en";
+        }
+    }
+
     if (preferredLocale == nil) {
```

Comment:
Is there a situation where preferred locale is passed from outside? Is there some case where someone would pass `nl` here and effectively override the changes you did because of the last block of this function?

Reply:
It is only set by effectiveLocale or changeLanguage from what I could figure